.gitignore

```diff
@@ -45,20 +47,8 @@ elements/**/*.d.ts
 core/**/*.d.ts
 tools/**/*.d.ts
 
-elements/**/__*.scss
-core/**/__*.scss
-tools/**/__*.scss
-
-elements/**/*.css
-core/**/*.css
 tools/**/*.css
```

Comment:
I think we should remove this line as well, right?

Reply:
good catch ty